QUICK_START.md

```diff
@@ -0,0 +1,160 @@
+_MIT License_
+_Copyright (c) 2017 MassChallenge, Inc._
+
+## Quickstart for Impact-API
+
+1. Install [Docker](https://docs.docker.com/docker-for-mac/)
+(download the stable channel).
+
+2. Start Docker from the Applications folder.
+
+3. Get the source code.  If you already have impact-api checked out,
+then in a Terminal window go to that directory and bring it up to date
+with:
+```
+cd <impact-api-directory>
+git checkout development
+git pull
+```
+
+Note that the impact-api code is currently designed to work with an
+instance of the MassChallenge Accelerate platform.  Specifically it
+expects an instance the Accelerate platform to manage a shared
+database.  This quick start quide does not require a running version
+of the Accelerate platform, but it does require an initial database
+dump which is normally part of the accelerate code base.
+
+If you don't have impact-api checked out, but you do have the
+accelerate code base checked out, checkout the impact-api code
+next to the accelerate directory with:
+```
+cd <accelerate-directory>/..
+git clone https://github.com/masschallenge/impact-api.git
+cd impact-api
+```
+
+If you don't have the accelerate code base checked out, you will
+need it to get an initial database.  If you need this, then contact
+engineering@masschallenge.org.
+
+4. Run tests.  From the impact-api source directoy run:
+```
+make build
+make test
+```
+
+This will take a while, but all tests should pass.  Specifically the
+end of the output should look something like:
+
+```
+Ran 102 tests in 14.845s
+
+OK
+Destroying test database for alias 'default'...
+```
+
+4. Start server
+
+```
+make dev
+```
+
+Once the process completes (it will take a little while and the last
+line should say "BUILD COMPLETE"), visit http://localhost:8000 in a
+browser to see the login screen.
+
+5. In a separate shell, load an accelerate database.  Assuming
+you have an up-to-date accelerate source code directory next to
+the impact-api directory, the following should work:
+
+```
+make dbload GZ_FILE=../accelerate/test_data/initial_schema.sql.gz
+```
+
+6. Create a new superuser account:
+
+```
+make superuser
+```
+
+This will ask you for an email address, full name, short name and a
+password.  For the purposes of this tutorial, we will assume the email
+address is "admin@example.com".
+
+7. Grant new account default access permissions
+
+```
+make grant-permissions PERMISSION_USER=admin@example.com PERMISSION_CLASSES=v0_clients,v1_clients
+```
+
+Respond to the questions with "yes" or "y".
```

Comment:
Maybe say "any confirmation prompts" instead of "questions"? (Mainly to cover the case where these classes have already been created.)